UsbAccess/UsbAccess.cpp

```diff
@@ -129,6 +129,47 @@ namespace Plugin {
                 mod_time = st.st_mtime;
             return mod_time;
         }
+
+        bool isAbsPath(const string& mountPath, const string& givenPath) {
+            if(givenPath.rfind(mountPath, 0) == 0)
+                return true;
+            else
+                return false;
```

Comment:
Pls check for trailing slashes in the path and trim them out before doing a find.

Reply:
The code already takes care of the trailing slashes, since getMounted() API doesn't return the trailing slash and the rfind() method gets succeed with or without trailing slash.